pages/_app.js

```diff
@@ -1,12 +1,21 @@
 import { ThemeProvider } from 'styled-components';
 import '../styles/globals.css';
 import { createGlobalStyle } from 'styled-components';
+import useHydration from '../hooks/useHydration';
+import Pre from '../component/Pre';
```

Comment:
maybe you can give the component "Pre" a name that describes it better since its purpose is not clear at first sight